src/Components/Ignitor/src/Ignitor.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsShippingPackage>true</IsShippingPackage>
```

Comment:
does this mean taht it ships on nuget.org? or just to our blob feed?

Reply:
Forgot to hit save :)